code/game/objects/items/circuitboards/computer_circuitboards.dm

```diff
@@ -507,20 +511,26 @@
 	name = "Express Supply Console"
 	build_path = /obj/machinery/computer/cargo/express
 
-/obj/item/circuitboard/computer/cargo/express/emag_act(mob/living/user)
+/obj/item/circuitboard/computer/cargo/express/emag_act(mob/user, obj/item/card/emag/emag_card)
 	if(!(obj_flags & EMAGGED))
 		contraband = TRUE
 		obj_flags |= EMAGGED
-		to_chat(user, span_notice("You change the routing protocols, allowing the Drop Pod to land anywhere on the station."))
+		if (user)
+			balloon_alert(user, "drop pods can now land anywhere")
+			to_chat(user, span_notice("You change the routing protocols, allowing the Drop Pod to land anywhere on the station."))
```

Comment:
etc etc
Generally balloon alerts are for things which are both short and ephemeral, while you'd use `to_chat` for things which are too long for a balloon alert or are important to be able to refer to later
`"drop pods can now land anywhere" is probably the upper edge of what I would want in a balloon alert 😅